test/gtest_endtest.h

```diff
@@ -0,0 +1,23 @@
+#ifndef GTEST_ENDTEST
+#define GTEST_ENDTEST
+
+
+#include <opencv2/core.hpp>
+#include <opencv2/videoio.hpp>
+
+#include <iostream>
+#include <stdlib.h>
+#include <thread>
+
+#include "imageProcessor.h"
+
+class EndTester : public imageProcessor{
```

Comment:
Does this just let the test view the frame received at the end of the chain? Can it not just grab it from the GUI instance?

Reply:
Yes, just it saves us relying on the gui when testing classes in isolation (and the gui always working)